ant-cli/src/commands/file.rs

```diff
@@ -278,6 +307,116 @@ async fn upload_dir(
     }
 }
 
+/// Uploads a file or directory to the network using Merkle batch payment.
+/// Single files are uploaded without an archive, directories are uploaded with an archive.
+/// The no_archive argument can be used to skip the archive upload.
+/// Returns the archive address if any and the address to access the data.
```

Comment:
seems in case of no-archive or multiple, a place holder string will be returned ?
```
Ok(("no-archive".to_string(), "multiple addresses".to_string()))
```
Which, better make the comment more accurate ?